README_EN.md

```diff
@@ -0,0 +1,879 @@
+# tinygo-keeb/workshop
+
+![](./images/zero-kb02.jpg)
+
+This page is for the TinyGo Keeb Tour that started on 2024/08/04.
+If you have any questions, please create in an issue in this repository or contact [twitter:sago35tk](https://x.com/sago35tk).
+
+For hardware assembly, please refer to the following.
+
+
+* [ビルドガイド](./buildguide.md)
+* [build guide (english)](./buildguide_EN.md)
+
+To this page QR code is here.  
+
+![](./images/qr_tinygo_keeb_workshop_top_english.png)
```

Comment:
Please also register the QR code below. You can download and register it as is.
![image](https://github.com/user-attachments/assets/8aa24b15-735e-4808-b659-357e2ba72b21)

Reply:
added the missing image